requirements-full.txt

```diff
@@ -1,2 +1,3 @@
 tifffile>=2018.10.18
 scanimage-tiff-reader==1.4.1
+neuroconv[video]
```

Comment:
Looks like you'll have to add `natsort` as well

Reply:
> Looks like you'll have to add `natsort` as well

You're right, thanks!

> Mostly looks good; is there an example file to share yet?

I've just uploaded at `tye-lab-to-nwb/C6-J588_Disc5_miniscope_stub.nwb`